examples/basic.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
we could do that, I mean, makes it easier to play around with when developing, might as well use the same for docs.